website/docs/docs/deploy/model-notifications.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Model notifications"
+description: "Receive notifications about failures with your models. "
+---
+
+# Model notifications <Lifecycle status="beta" />
+
+Set up dbt to notify the appropriate owners &mdash; through Slack, email, or both &mdash; about any failures your models have encountered. When models fail, your reports are no longer accurate and the data can no longer be trusted so it's important to resolve these failures quickly.
+
+To be timely and keep the number of notifications to a reasonable amount, dbt observes the following guidelines when notifying the model owners:  
+
+- For the first model that fails, send a notification to each unique model owner. This means that each owner receives only one notification, the initial one.
```

Comment:
Also, @rviswanathan-dbt  will need to weigh in on the rules here, but I think it's something like, one notification per unique owner for each thing:
e.g. one for model failure
one for test warning
one for test failure

But I might be getting this slightly wrong.

Reply:
The rule is 1 notification per unique owner/email during a run for any of the following -
failure/success/warning of tests
failure/success of models (there is no model warning)

At the end of the run, the owner will get an email for each of the statuses they've wanted to be notified for, with the list of models/tests that have that status

Reply:
thank you! folded this in. please lemme know if i didn't capture this ^ correctly

Reply:
yes, you've captured it correctly. Thanks Ly!